db/schema.rb

```diff
@@ -323,6 +323,24 @@
     t.index ["unique_id", "facility_type"], name: "index_base_facilities_on_unique_id_and_facility_type", unique: true
   end
 
+  create_table "benefits_intake_form_submissions", force: :cascade do |t|
+    t.string "form_type", null: false
+    t.uuid "benefits_intake_uuid"
+    t.uuid "submitted_claim_uuid"
+    t.jsonb "form_data", default: {}
+    t.uuid "user_account_id"
+    t.bigint "saved_claim_id"
+    t.bigint "in_progress_form_id"
```

Comment:
what was the use case for 'submission' of InProgressForm - should this also be dropped/moved to a separate table